src/components/views/rooms/RoomHeader.tsx

```diff
@@ -112,6 +120,18 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
     const threadNotifications = useRoomThreadNotifications(room);
     const globalNotificationState = useGlobalNotificationState();
 
+    const directRoomsList = useAccountData<Record<string, string[]>>(client, EventType.Direct);
+    const [isDirectMessage, setDirectMessage] = useState(false);
+    useEffect(() => {
+        for (const [, dmRoomList] of Object.entries(directRoomsList)) {
+            if (dmRoomList.includes(room?.roomId ?? "")) {
```

Comment:
```suggestion
            if (room?.roomId != "" && dmRoomList.includes(room.roomId)) {
```
(Or something nicer?)